src/lib/db/feature-toggle-client-store.ts

```diff
@@ -218,6 +210,22 @@ export default class FeatureToggleClientStore
             ?.constraints.push(...row.segment_constraints);
     }
 
+    private addSegmentIdsToStrategy(
+        feature: PartialDeep<IFeatureToggleClient>,
+        row: Record<string, any>,
+    ) {
+        const strategy = feature.strategies.find(
+            (s) => s.id === row.strategy_id,
+        );
+        if (!strategy) {
+            return;
+        }
+        if (strategy.segments == null) {
```

Comment:
Could do `if (!strategy.segments) {` to avoid the double equals.

Reply:
Good spot, thanks, yep that's definitely better